### DISCUSSION TITLE
0개의 리스트에서 잔여백신 시도중

### DISCUSSION THREAD

**Discussion by adjoj** [kimytsc/covid-rest-vaccine-macro#38] [Q&A] [answered]
화이자로 설정 해놓았는데 이렇게 뜨네요. 제가 잘못 하고 있는 건가요? 아니면 근처에 화이자 보유 병원이 없어서 이렇게 뜨는 건가요?

**Comment by kimytsc** [comment, marked as answer]
0개의 리스트라면 일시적으로 차단되었거나 지도 설정이 잘못된 경우로 판단됩니다.

확인을 위해 설정하신 map url이나 bounds 전달 가능할까요?

**Comment by adjoj** [comment]
주소창에 붙여놓은 url 말씀하신거 맞죠?
javascript:((my={
    map: "https://m.place.naver.com/rest/vaccine?vaccineFilter=unused&x=129.0842616&y=35.1987824&bounds=129.0513026%3B35.1873142%3B129.1172206%3B35.210249",
    delay: 1000,
    timeout: 3000,
    choice: [
        "VEN00013",/*화이자*/
           ]
}) => {
fetch('https://raw.githubusercontent.com/kimytsc/covid-rest-vaccine-macro/main/kakao/macro.js')
.then(res => res.text())
.then(res => {
    var d=document
    , s=d.createElement('script');
    Object.keys(my).forEach(k=>{s.setAttribute(k, my[k])});
    s.innerHTML=res;
    d.getElementsByTagName('head')[0].appendChild(s);
});
})()

여기요!

**Reply by kimytsc** [reply]
확인해보니 95개 리스트가 나오는데, 일시적으로 차단되었을 가능성이 높아 보입니다.

일시적인 차단으로 카카오가 안될 경우, 네이버쪽으로 시도해보시면 좋을 것 같습니다.

**Comment by adjoj** [comment]
네 감사합니다!!!
